library/entropy.c

```diff
@@ -285,87 +285,88 @@ int mbedtls_entropy_gather( mbedtls_entropy_context *ctx )
 
 int mbedtls_entropy_func( void *data, unsigned char *output, size_t len )
```

Comment:
Please note that this PR changes the behaviour of the function, but it does not update the doxygen documentation for `mbedtls_entropy_func()` that still says:
```
 * \param len       Number of bytes desired, must be at most MBEDTLS_ENTROPY_BLOCK_SIZE
```